static/app/views/settings/organizationIntegrations/reinstallAlert.tsx

```diff
@@ -0,0 +1,29 @@
+import {ReactNode} from 'react';
+
+import {Alert} from 'sentry/components/alert';
+import {t} from 'sentry/locale';
+import {Integration} from 'sentry/types';
+import {getIntegrationStatus} from 'sentry/utils/integrationUtil';
+
+type Props = React.ComponentPropsWithoutRef<typeof Alert> & {
```

Comment:
kinda same here, might not make sense to add all the alert props if there's no plans on displaying more than one thing